.changeset/gold-badgers-float.md

```diff
@@ -2,21 +2,23 @@
 "@fluidframework/shared-object-base": minor
 ---
 
-Added typeguard for SharedObjectKinds
+New feature: Added type guards for DDS types
```

Comment:
This might be a challenge if we move to automatically adding package names as prefix to change titles. I like this better than having `shared-object-base` here.

Reply:
This sort of thing also seems like a good place to leverage conventional commits. I.e., derive this automatically from the PR CC tag (if present)

Reply:
Or just use a similar syntax in these files. Either works. I'm still a fan of CC for other reasons, but so long as we have a consistent pattern, I'm a happy camper 🙂